product_unique_serial/model/stock.py

```diff
@@ -54,78 +54,25 @@ def _get_last_location_id(self):
-                                           'number'))
-            # Customize domain
-            domain_quants = [
-                ('product_id', '=', move.product_id.id),
-                ('lot_id', '=', lot_id),
-                ('qty', '>', 0.0)
-            ]
-            # Check product tracking field to get location usage for domain
-            if move.product_id.track_all:
-                domain_quants += [('location_id.usage', '!=', 'inventory')]
-            else:
-                usages = []
-                if move.product_id.track_incoming:
-                    usages += ['internal']
-                if move.product_id.track_outgoing:
-                    usages += ['customer', 'transit']
-                # mrp module should be installed to use track production field
-                if hasattr(move.product_id, "track_production") and \
-                        move.product_id.track_production:
-                    usages += ['production']
-                if usages:
-                    domain_quants += [
-                        ('location_id.usage', 'in', tuple(usages))]
-            # check if exist other similar quant
-            if self.search(domain_quants):
-                lot_name = self.env['stock.production.lot'].browse(lot_id).name
-                raise exceptions.Warning(_('The serial number %s can only '
-                                           'belong to a single product in '
-                                           'stock') % lot_name)
-        return super(StockQuant, self)._quant_create(
-            qty, move, lot_id, owner_id, src_package_id,
-            dest_package_id, force_location_from, force_location_to)
-
     @api.multi
     @api.constrains('product_id', 'lot_id', 'qty')
     def _check_inicity_lot_product(self):
         for line in self:
             if line.lot_id and line.product_id and\
                     line.product_id.lot_unique_ok:
-                note = _(
-                    'When you select a serial number (lot), the quantity is '
-                    'corrected with respect to\nthe quantity of that serial '
-                    'number (lot) and not to the total quantity of the '
-                    'product.')
                 quants = self.search([
                     ('lot_id', '=', line.lot_id.id),
                     ('product_id', '=', line.product_id.id),
                     ('company_id', '=', line.company_id.id),
                     ('location_id', '=', line.location_id.id)])
                 if line.qty > 1:
                     raise ValidationError(_(
-                        'The product %s has active unique lot and you '
-                        'try set %s in the lot %s.\nRemmember:\n %s' % (
-                            line.product_id.name, line.qty,
-                            line.lot_id.name, note)))
+                        'You should only receive by the piece with the same '
+                        'serial number'))
```

Comment:
I am using the same lines that were used before,

https://github.com/Vauxoo/stock-logistics-workflow/pull/14/files#diff-740d05815bfc94b847d7e0d9d485423fL94

In order to be too much hassle the module's test itself and other test upwards